### DISCUSSION TITLE
Solicitud rechazada

### DISCUSSION THREAD

**Discussion by amsm30-ua** [community/community#138385] [GitHub Education]
### Hi everyone,

La página de Github no me deja utilizar la camara de mi PC cosa que no entiendo porque ya que tiene todos los permisos habilitados, pruebo con el telefono movil y tampoco me deja usar la camara trasera, despues de haber hecho la foto con la camara frontal (que ya ha sido dificil hacerla) me mandan un correo instantaneamente con que me han rechazado la solicitud.
¿Qué puedo hacer?

**Comment by itisrude** [comment]
Hola amsm30-ua,

Eso es realmente frustrante para usted. Suponiendo que tienes todos los permisos activados y aún así no te otorga acceso a la cámara de la PC o a la parte posterior de tu teléfono, aquí hay un par de cosas que puedes probar:

- Intente tomar una captura de pantalla del documento en su computadora en lugar de tomar una foto con su teléfono. Eso debería evitar los problemas con la cámara.

- Verificación del formato del archivo: Incluso en esta etapa, después de tomar la foto, el sistema puede rechazar el formato. Asegúrese de que sea lo más claro posible.

- Soporte de GitHub: si ha intentado todo y aún así lo rechazan, es posible que desee enviarles una nota al soporte, explicando lo que está pasando. Ellos pueden revisar su caso con mayor detalle.

¡Espero que ayude!
